app/src/screens/BankDetalisScreen.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { SafeAreaView, Text } from "react-native";
```

Comment:
```suggestion
import { Text } from "react-native";
import { SafeAreaView } from 'react-native-safe-area-context';
```
use this to avoid status bar